src/main/java/io/snowdrop/github/issues/Assignees.java

```diff
@@ -0,0 +1,38 @@
+
+package io.snowdrop.github.issues;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Assignees implements Serializable {
```

Comment:
Is there any reason why this need to be `Serializable`?

Reply:
To rhyme with the rest of the egit-github domain objects. I don't know if it HAS too.